lib/common-dependencies.gradle

```diff
@@ -152,34 +210,34 @@ configure(rootProject) {
             group: 'Build',
             description: 'Generates the file that contains dependency versions.') {
 
-        inputs.property("catalog", project.extensions.getByType(VersionCatalogsExtension))
         def f = file("${project.buildDir}/managed_versions.yml")
         outputs.file(f)
 
         doLast {
             f.parentFile.mkdir()
-            managedVersions = [:]
-            inputs.properties["catalog"].catalogNames.forEach { name ->
+            def managedVersions = [:]
+            def catalogs = project.extensions.getByType(VersionCatalogsExtension)
```

Comment:
This was changed by https://github.com/line/centraldogma/pull/716/files#diff-99c9f9c14309adeee494c305b31bb4774d339f731c426330858d51e3190a0f30R160